src/astroport/robot.rs

```diff
@@ -355,6 +355,19 @@ mod tests {
         ContractMap, TestRunner,
     };
 
+    use crate::traits::CwItRunner;
+
+    use cosmwasm_std::coin;
+
+    fn initial_coins() -> Vec<cosmwasm_std::Coin> {
+        vec![
+            coin(u128::MAX, "uosmo"),
+            coin(u128::MAX, "uion"),
+            coin(u128::MAX, "uatom"),
+            coin(u128::MAX, "stake"),
+        ]
+    }
```

Comment:
why this fn once here and once in utils?

Reply:
fixed